src/libcollections/btree/set.rs

```diff
@@ -11,12 +11,12 @@
 // This is pretty much entirely stolen from TreeSet, since BTreeMap has an identical interface
 // to TreeMap
 
-use core::cmp::Ordering::{self, Less, Greater, Equal};
-use core::cmp::{min, max};
+use core::cmp::Ordering::{self, Equal, Greater, Less};
+use core::cmp::{max, min};
```

Comment:
See also the discussion and my proposal [here](https://github.com/rust-lang/rust/pull/34085#discussion_r65827942) for a more general version of this rule.